great_expectations/core/expectation_configuration.py

```diff
@@ -93,13 +105,19 @@ def make_expectation_context(self, data, **kwargs):
         return ExpectationContext(**data)
 
 
+class KWargDetailsDict(TypedDict):
+    domain_kwargs: tuple[str, ...]
+    success_kwargs: tuple[str, ...]
+    default_kwarg_values: dict[str, str | bool | float | RuleBasedProfilerConfig | None]
+
+
 class ExpectationConfiguration(SerializableDictDot):
     """ExpectationConfiguration defines the parameters and name of a specific expectation."""
 
-    kwarg_lookup_dict = {
+    kwarg_lookup_dict: ClassVar[Mapping[str, KWargDetailsDict]] = {
```

Comment:
I was expecting making the tuple changes below to be a nightmare because missing one change would cause failures. Thankfully mypy was able to point to the specific lines of each specific nested dict instead of an unhelpful error on line `117` that the whole object was wrong.